packages/@aws-cdk/pipelines/README.md

```diff
@@ -27,7 +27,11 @@ same, the *CDK Pipelines* library takes care of the details.
 will work, see the section **CDK Environment Bootstrapping** below).
 
 ```ts
-import { Construct, Stage } from '@aws-cdk/core';
+import { DatabaseStack, ComputeStack } from '../lib/my-stacks';
```

Comment:
do we need to provide a couple sentences to introduce these stacks and what they contain?
Adding the import provides more of a clue of how the app is structured. However, it does not quite bridge the gap of understanding what they are / what they contain.

Reply:
I've added some comments.